test/yml_SDP/CatLibrary.Cat-2.ownEat.yml

```diff
@@ -28,6 +28,13 @@ members:
   - cat-1.0
   - cat-2.0
   summary: Eat event of this cat
+   remarks: >-
+    This should not be changed.
+
+    Not this line.
+
+
+    Nor this one.
   metadata: {}
```

Comment:
Here, again, this spacing was performed by the existing logic in `NormalizeDocsElement` but looks off to me based on the source and XML:

```xml
        <remarks>
        <format>
          This should not be changed.
          Not this line.

          Nor this one.
        </format>
        </remarks>
```

Reply:
Similar to the last one, this is just caused by yaml serializing style.